FBSDKLoginKit/FBSDKLoginKit/FBSDKLoginButton.m

```diff
@@ -105,6 +105,7 @@ - (CGSize)sizeThatFits:(CGSize)size
 
 #pragma mark - UIActionSheetDelegate
 
+#pragma clang diagnostic ignored "-Wdeprecated-declarations"
```

Comment:
Can you surround the statement with `#pragma clang diagnostic push` and `#pragma clang diagnostic pop` so that it doesn't suppress other code?